core/src/main/java/org/apache/calcite/runtime/JsonFunctions.java

```diff
@@ -791,6 +912,38 @@ public boolean hasException() {
     UNKNOWN,
     NONE
   }
+
+  /**
+   * Used in the JsonModify function.
+   */
+  public enum JsonModifyMode {
+    REPLACE("replace"),
+
+    INSERT("insert"),
+
+    SET("set"),
+
+    REMOVE("remove");
+
+    String type;
+
+    JsonModifyMode(String type) {
+      this.type = type;
+    }
+
+    public String getType() {
```

Comment:
We can remove this method (and the other `getType` method), then use the Enum instance directly.